client/components/vertical-menu/style.scss

```diff
@@ -41,3 +49,6 @@
 	margin-bottom: -4px;
 }
 
+.vertical-menu__items__social-icon .social-logo {
+	fill: $gray;
```

Comment:
That's just for aesthetics. If the fill color was the same as the text color, the icons would look a little bit too dark.

![screen shot 2017-02-13 at 12 41 46 pm](https://cloud.githubusercontent.com/assets/618551/22897582/dfa846ea-f1e9-11e6-8ed4-a2ac45249a72.png)